### PR TITLE
Make partest defaults quieter.

### PR DESCRIPTION
**Pull request by paulp**
Only show failure transcipts under --verbose. Only show individual test
successes under --verbose. Regular verbosity gets a . for each success and a
straight list of test failures at the end of the run.

Chased down a couple other partest annoyances, such as partest-ack counting
(and trying to run) tests more than once. Also cleaned up the orphan check and
flag files people leave lying around when the move tests aside.

### CODE REVIEW

**Comment by xeno-by** [comment]
Thank you Paul!

**Comment by xeno-by** [comment]
Some questions from looking at https://scala-webapps.epfl.ch/jenkins/job/pr-rangepos-per-commit/2449/console as an example:
1) Jenkins no longer needs "read more" links to display its logs. Yay!!
2) It looks that not all tests have dots upon success. Some of them (e.g. `scalap` and `scalacheck`) are silent.
3) Also it looks like by default partest does display failure diffs. This is not what is advertised in the pull request mesage.
4) Thank you for leaving error messages in. Incremental analysis is important!
5) I wonder why everywhere we have `!!` creating a new line in the report, but in `[partest] .............................................!!   2 - neg/t800.scala                            [output differs]` we do not.

File: src/reflect/scala/reflect/internal/util/Collections.scala [hunk 1]
```diff
@@ -209,3 +209,5 @@ trait Collections {
     case _: IllegalArgumentException => None
   }
 }
+
+object Collections extends Collections
```
Comment (xeno-by) [inline src/reflect/scala/reflect/internal/util/Collections.scala:213]: I wonder why this got removed at some point.

Comment (paulp) [inline src/reflect/scala/reflect/internal/util/Collections.scala:213]: When I did my sweep of dead code, I was pretty aggressive. Some things which were plausibly useful, just not yet being used, were included.

**Comment by soc** [comment]
If we make terse the default, can we increase the verbosity of the verbose one?
E. g. print which test is started before running it, so it is easier to diagnose which test is hanging.

**Comment by paulp** [comment]
Reopening.

**Pull request closed without merging**